src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -613,9 +629,11 @@ void Creature::Update(uint32 diff)
 
                 if (CanNotReachTarget() && !IsInEvadeMode() && !GetMap()->IsRaid())
                 {
+                    sLog->outString("1 EVADE?");
```

Comment:
clean (?)

Reply:
Debug lines, all will be removed once it's tested and verified working, haven't cleaned it up yet.